Sources/SimpleCurrencyConverter/DependencyInjection/DependencyContainer.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+class DependencyContainer: DependencyProtocol {
```

Comment:
@ts-dzmitry-kurski Please declare `DependencyContainer` as `final class` if inheritance is not needed.